src/modules/formBuilder/components/FormEditorItemProperties.tsx

```diff
@@ -0,0 +1,182 @@
+import { FormItem } from '@/types/gqlTypes';
+
+const dataCollectedAboutPickList =
+  localResolvePickList('DataCollectedAbout') || [];
+const disabledDisplayPickList = localResolvePickList('DisabledDisplay') || [];
+const inputSizePickList = localResolvePickList('InputSize') || [];
+
+interface FormEditorItemPropertiesProps {
+  item: FormItem;
+  onChangeProperty: (attr: keyof FormItem, newProperty: any) => void;
+}
+
+const FormEditorItemProperties: React.FC<FormEditorItemPropertiesProps> = ({
+  item,
+  onChangeProperty,
+}) => {
+  return (
+    <>
+      <Typography>Properties</Typography>
+      <Stack
+        gap={2}
+        component='form'
+        onSubmit={() => {
+          console.log('submitted'); // TODO(#6103)
+        }}
+        sx={{ maxWidth: MAX_INPUT_AND_LABEL_WIDTH }}
+      >
+        <TextInput
+          label='Link ID'
+          value={item.linkId}
+          onChange={(e) => {
+            onChangeProperty('linkId', e.target.value);
+          }}
+          helperText='Unique ID for this form item'
+        />
+        <LabeledCheckbox
+          label='Assessment date'
+          checked={!!item.assessmentDate}
+          onChange={(e) =>
+            onChangeProperty(
+              'assessmentDate',
+              (e.target as HTMLInputElement).checked
+            )
+          }
+        />
+        <LabeledCheckbox
+          label='Required'
+          checked={item.required}
+          onChange={(e) =>
+            onChangeProperty('required', (e.target as HTMLInputElement).checked)
+          }
+        />
+        <TextInput
+          label='Label'
+          value={item.text}
+          onChange={(e) => {
+            onChangeProperty('text', e.target.value);
+          }}
+        />
+        <TextInput
+          label='Helper text'
+          value={item.helperText || ''}
+          onChange={(e) => {
+            onChangeProperty('helperText', e.target.value);
+          }}
+        />
+        <TextInput
+          label='Brief label'
+          value={item.briefText || ''}
+          onChange={(e) => {
+            onChangeProperty('briefText', e.target.value);
+          }}
+          helperText="Label to display when the item is referenced briefly, such as in an Autofill dialog box. If not specified, the item's normal label text is shown."
+        />
+        <TextInput
+          label='Read-only label'
+          value={item.readonlyText || ''}
+          onChange={(e) => {
+            onChangeProperty('readonlyText', e.target.value);
+          }}
+          helperText="Label to display when the item is shown in a read-only form. If not specified, the item's normal label text is shown."
+        />
+        <LabeledCheckbox
+          label='Read-only'
+          checked={item.readOnly}
+          onChange={(e) =>
+            onChangeProperty('readOnly', (e.target as HTMLInputElement).checked)
+          }
+        />
+        <LabeledCheckbox
+          label='Hidden'
```

Comment:
I was a bit puzzled over what the Preview should display if the Hidden attribute is true. To discuss with design?